doc/source/timeseries.rst

```diff
@@ -185,6 +185,13 @@ options like ``dayfirst`` or ``format``, so use ``to_datetime`` if these are req
 
     pd.Timestamp('2010/11/12')
 
+You can also use the `DatetimeIndex` constructor directly:
+In [3]: pd.DatetimeIndex(['2018-01-01', '2018-01-03', '2018-01-05'])
```

Comment:
Can you put this in a `.. ipython:: python` block? (same procedure as in my previous comment)